docs/sources/trackers/google-tag-manager/snowplow-template/plugins/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Plugins"
+sidebar_position: 200
+---
+
+Plugins provide an easy way to distribute custom behavior for the tracker.
+Each Plugin can be loaded from external URLs or provided inline within GTM, and can optionally take some custom configuration.
+A Plugin may add new methods to the tracker that can be called via [Custom Commands](#custom-commands).
+
+:::note
+
+Snowplow Plugins are updated alongside the tracker SDK to ensure compatibility. For best results, keep the SDK and plugin versions in sync:
```

Comment:
which tracker sdk?

Reply:
On second though, SDK probably isn't even a good term to use here, it's a bit more developer-ey than I was going for with this update. I've replaced it with `tracker` where appropriate.